apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/mechanism_apic.py

```diff
@@ -916,6 +927,13 @@ def create_subnet_precommit(self, context):
         subnet = context.current
         network = context._plugin.get_network(context._plugin_context,
                                               subnet['network_id'])
+        context.subnet_scope = self.net_cons.get_subnet_scope(
+            str(self.name_mapper.tenant(context, network['tenant_id'])),
+            network['name'],
```

Comment:
If I understand correctly, the subnet scope is defined by tenant/network. A network however is a runtime resource, as opposed to the net-cons file that is constant... Are we limiting the use case of the subnet scope to APIC managed external networks? (the only kind of network I can think of that has a predictable name)